rclcpp/include/rclcpp/subscription_base.hpp

```diff
@@ -263,6 +263,41 @@ class SubscriptionBase : public std::enable_shared_from_this<SubscriptionBase>
   bool
   exchange_in_use_by_wait_set_state(void * pointer_to_subscription_part, bool in_use_state);
 
+  /// Check if subscription instance can support content filter topic feature.
+  /**
+   * Depending on the middleware and the message type, this will return true if the middleware
```

Comment:
oversight from internal review, is this dependent on message type?

Reply:
Thanks for pointing it out.